libcxx/test/support/test_range.h

```diff
@@ -10,9 +10,11 @@
 #define LIBCXX_TEST_SUPPORT_TEST_RANGE_H
 
 #include <concepts>
+#include <functional>
 #include <iterator>
 #include <ranges>
 
+#include "__concepts/invocable.h"
```

Comment:
We can't include private libc++ headers in the test suite since the test suite must work with other implementations too. Also I think this header is not necessary anyway.